js-glaredb/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+tab_spaces = 2
```

Comment:
Could we remove this?

Reply:
oh yeah, that was auto generated from napi.